nmagent/client_test.go

```diff
@@ -432,3 +432,73 @@ func TestNMAgentDeleteNC(t *testing.T) {
 		})
 	}
 }
+
+func TestNMAgentSupportedAPIs(t *testing.T) {
+	tests := []struct {
+		name      string
+		exp       nmagent.HomeAzInfo
+		expPath   string
+		resp      map[string]interface{}
+		shouldErr bool
+	}{
+		{
+			"happy path",
+			nmagent.HomeAzInfo{HomeAz: "az01"},
+			"/machine/plugins/?comp=nmagent&type=GetHomeAzInfo",
+			map[string]interface{}{
+				"httpStatusCode": "200",
+				"HomeAz":         "az01",
+				"DcmtRegion":     "uswest2",
```

Comment:
we won't be getting dcmtregion anymore. HomeAz returned would be 1 or 01. Let;s ask for the doc from Alok to finalize these values.